lambdas/dynamo-unmarshall/src/dynamo-unmarshall-handler.ts

```diff
@@ -0,0 +1,70 @@
+import { MetricUnit } from "@aws-lambda-powertools/metrics";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { LambdaInterface } from "@aws-lambda-powertools/commons/types";
+import { unmarshall } from "@aws-sdk/util-dynamodb";
+
+import {
+  HandlerMetricExport,
+  MetricsProbe,
+} from "../../../lib/src/Service/metrics-probe";
+
+import {
+  HandlerMetric,
+  CompletionStatus,
+} from "../../../lib/src/MetricTypes/handler-metric-types";
+
+const logger = new Logger({ serviceName: "DynamoUnmarshallHandler" });
+
+export class DynamoUnmarshallHandler implements LambdaInterface {
+  metricsProbe: MetricsProbe;
+
+  constructor(metricsProbe: MetricsProbe) {
+    this.metricsProbe = metricsProbe;
+  }
+  @logger.injectLambdaContext({ clearState: true })
+  @HandlerMetricExport.logMetrics({
+    throwOnEmptyMetrics: false,
+    captureColdStartMetric: true,
+  })
+  public async handler(event: any, _context: unknown): Promise<object> {
+    try {
+      logger.debug(JSON.stringify(event));
```

Comment:
We should remove this as we could end up logging PII in prod if the log level gets changed